src/vacancies/vacancies-router.js

```diff
@@ -63,6 +63,20 @@ vacancyRouter
         return res.status(400).json({ error: `${skills} ${wrongSkills}` });
       }
 
+      const tagsLimit = VacancyService.validateTags(skills);
+      if (tagsLimit) {
+        return res.status(400).json({ error: `${skills} ${tagsLimit}` });
+      }
+
+      for (var i = 0; i < skills.length; i++) {
+        const tagsLimitLength = VacancyService.validateTagLength(skills[i]);
+        if (tagsLimitLength) {
+          return res
+            .status(400)
+            .json({ error: `${skills[i]} ${tagsLimitLength}` });
+        }
+      }
```

Comment:
Let's use a forEach, since we're looping over the entire array anyway. Also, we do not want to use `var` to declare a variable unless we have good reason to do so. 

Something like: 

```suggestion
      skills.forEach(skill => {
          const tagLengthError = VacancyService.validateTagLength(skill);
          if (tagLengthError) {
              return res
                  .status(400)
                  .json({ error: `${skill} ${tagLengthError}` })
          }
      })
```